samples/philosophers/sample.yaml

```diff
@@ -3,4 +3,13 @@ sample:
 tests:
   test:
     tags: samples
+    extra_args: "-DDEBUG_PRINTF=1"
     harness: console
+    harness_config:
+      type: multi_line
+      repeat: 10
+      regex:
+        - ".*STARVING.*"
```

Comment:
do the regexes need to be in this order or does it pass if it encounters any of these strings in indeterminate order?